src/libs/actions/Report.js

```diff
@@ -166,7 +166,7 @@ function getSimplifiedReportObject(report) {
     // We convert the line-breaks in html to space ' ' before striping the tags
     const lastMessageText = lodashGet(lastReportAction, ['message', 'html'], '')
         .replace(/((<br[^>]*>)+)/gi, ' ')
-        .replace(/(<([^>]+)>)/gi, '');
+        .replace(/(<([^>]+)>)/gi, '') || `[${translateLocal('common.deletedCommentMessage')}]`;
```

Comment:
hmmmm, can you make sure to escape any character that has a meaning in regex? Right now there's no risk, but it would be very easy for a translation to include a dot or some other character that regexes would treat differently than just a text (same below).

Reply:
I'm sorry I'm not following. The translated text is not checked against any regular expression and isn't treated as a regular expression. I'm not sure what's the concern is here. Can you please rephrase it somehow maybe?

Reply:
Oh sorry, you are right, I misread this.